ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -51022,6 +51022,13 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-ovn-upgrade-rollback-oldest-supported
+  reporter_config:
+    slack:
+      channel: '#ocp-qe-rollback-ota'
```

Comment:
@jianlinliu Are you good with the channel name or the pattern with channel w/o ping? If yes, I'll ask somebody to approve:)

Reply:
how about `ocp-rollback-ci`?

Reply:
sounds okay for me.